.expeditor/config.yml

```diff
@@ -9,31 +9,34 @@ pipelines:
       env:
         - ADHOC: true
 
-merge_actions:
-  - built_in:bump_version:
-      ignore_labels:
-        - "Version: Skip Bump"
-        - "Expeditor: Skip All"
-  - built_in:update_changelog:
-      ignore_labels:
-        - "Changelog: Skip Update"
-        - "Expeditor: Skip All"
-  - trigger_pipeline:omnibus/release:
-      ignore_labels:
-        - "Omnibus: Skip Build"
-        - "Expeditor: Skip All"
-      only_if: built_in:bump_version
+subscriptions:
+  - workload: pull_request_merged:{{github_repo}}:{{release_branch}}:*
+    actions:
+      - built_in:bump_version:
+          ignore_labels:
+            - "Version: Skip Bump"
+            - "Expeditor: Skip All"
+      - built_in:update_changelog:
+          ignore_labels:
+            - "Changelog: Skip Update"
+            - "Expeditor: Skip All"
+      - trigger_pipeline:omnibus/release:
+          ignore_labels:
+            - "Omnibus: Skip Build"
+            - "Expeditor: Skip All"
+          only_if: built_in:bump_version
+  
+  - workload: project_promoted:{{agent_id}}:*
+    actions:
+      - built_in:promote_artifactory_artifact
 
 artifact_actions:
```

Comment:
Wow, this is actually a super outdated config. We should remove it and add the following subscription. 

```
- workload: artifact_published:stable:marketplace:*
  actions:
    - built_in:rollover_changelog
    - built_in:create_github_release
    - built_in:notify_chefio_slack_channels
```

Reply:
made the changes as suggested